data/base-hunting.yaml

```diff
@@ -745,20 +745,25 @@ hunting_zones:
   - 8702
   - 8703
   # https://elanthipedia.play.net/Orc_raider 365-450
+  # downstairs mixed with reivers
   orc_raiders:
   - 8704
   - 8705
   - 8706
   - 8707
   - 8708
   - 8709
+  # upstairs just raiders
   orc_raiders2:
   - 8710
   - 8711
   - 8712
   - 8713
   - 8714
   - 8715
+  # https://elanthipedia.play.net/Orc_clan-chief
```

Comment:
Could you add the skill range to this comment?

Reply:
Added and adjusted comments.